common/djangoapps/util/enterprise_helpers.py

```diff
@@ -0,0 +1,125 @@
+"""
+Helpers to access the enterprise app
+"""
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+try:
+    from enterprise.models import EnterpriseCustomer
+    from enterprise.tpa_pipeline import (
+        active_provider_requests_data_sharing,
+        active_provider_enforces_data_sharing,
+        get_enterprise_customer_for_request,
+    )
+
+except ImportError:
+    pass
+
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+
+def enterprise_enabled():
+    """
+    Determines whether the Enterprise app is installed
+    """
+    return 'enterprise' in settings.INSTALLED_APPS
+
+
+def data_sharing_consent_requested(request):
+    """
+    Determine if the EnterpriseCustomer for a given HTTP request
+    requests data sharing consent
+    """
+    if enterprise_enabled():
+        return active_provider_requests_data_sharing(request)
+    else:
+        return False
+
+
+def data_sharing_consent_required_at_login(request):
+    """
+    Determines if data sharing consent is required at
+    a given location
+    """
+    if enterprise_enabled():
+        return active_provider_enforces_data_sharing(request, EnterpriseCustomer.AT_LOGIN)
+    else:
+        return False
+
+
+def data_sharing_consent_requirement_at_login(request):
+    """
+    Returns either 'optional' or 'required' based on where we are.
+    """
+    if not enterprise_enabled():
+        return None
+    if data_sharing_consent_required_at_login(request):
+        return 'required'
+    if data_sharing_consent_requested(request):
+        return 'optional'
+    return None
+
+
+def insert_enterprise_fields(request, form_desc):
+    """
+    For right now, a simple wrapper for `add_data_sharing_consent_field`,
```

Comment:
What is meant by "For right now"?  How long of a time frame are we talking about?  Let's be more specific about what we are doing here and why.